reference/index.rst

```diff
@@ -2,20 +2,10 @@ Reference
 =========
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
 
-
-.. toctree::
-   :caption: Other Useful Documentation
-
-   Accessing CloudOps' Jenkins <https://github.com/mozilla-services/cloudops-deployment#accessing-jenkins>
-   Balrog <https://mozilla-balrog.readthedocs.io/en/latest/index.html>
-   CloudOps Contact Info <https://mana.mozilla.org/wiki/display/SRE/Contacting+Services+SRE>
-   Firefox L10n <https://firefox-source-docs.mozilla.org/l10n/index.html>
-   Partner Repacks <https://firefox-source-docs.mozilla.org/taskcluster/partner-repacks.html>
-   Releng Mana <https://mana.mozilla.org/wiki/display/RelEng/Release+Engineering+Home>
-   Scriptworker <https://scriptworker.readthedocs.io/en/latest/index.html>
-   Scriptworker Scripts <https://scriptworker-scripts.readthedocs.io/en/latest/index.html>
-   Scriptworker-scripts Wiki (mac signing procedures) <https://github.com/mozilla-releng/scriptworker-scripts/wiki>
-   Taskgraph <https://taskcluster-taskgraph.readthedocs.io/>
-   Treeherder <https://treeherder.readthedocs.io/>
+   access
+   machine-users
+   logs
+   /releng_changelog
```

Comment:
Is it time to do away with the changelog? If not, we should start using this again :)

Reply:
A very good question. Personally I've never found value in it even when we actually used it somewhat. This is probably worth its own bullet point at the team meeting.